### PR TITLE
Margin on surveys is making it overlap with sections on top of it

### PR DESCRIPTION
**Pull request by sewagodimo**
A -15% margin value is added when the survey list only has on survey however this causes an overlap on the sections list because there is content above the survey even though it's still the only survey on the page.
Removing the margin value removes the overlaping values

### CODE REVIEW

**Comment by sewagodimo** [comment]
image

**Comment by Mitso** [comment]
@sewagodimo Can you promote a banner on section.

**Comment by sewagodimo** [comment]
@Mitso this is the homepage with the banner and the survey
image

**Comment by sewagodimo** [comment]
Fixed the actual issue by giving the sections nav a bottom margin
image

**Comment by sewagodimo** [comment]
The view without the banner
image

File: gem/templates/rwanda/core/section_page.html [hunk 1]
```diff
@@ -137,7 +139,7 @@ <h2 class="section-nav__label">
         {% endif %}
     {% endfor %}
   </ul>
-</div>
+  </div>
```
Comment (sewagodimo) [inline gem/templates/rwanda/core/section_page.html]:
```suggestion
</div>
```

File: gem/templates/rwanda/core/section_page.html [hunk 2]
```diff
@@ -118,7 +118,9 @@ <h2 class=" heading__component">
       {% trans "Article categories" %}
     </h2>
   </div>
-  {% load_sections as sections %}
+</div>
+
```
Comment (sewagodimo) [inline gem/templates/rwanda/core/section_page.html]:
```suggestion
```

File: gem/templates/rwanda/core/section_page.html [hunk 3]
```diff
@@ -118,7 +118,8 @@ <h2 class=" heading__component">
       {% trans "Article categories" %}
     </h2>
   </div>
-  {% load_sections as sections %}
+</div>
+{% load_sections as sections %}
```
Comment (sewagodimo) [inline gem/templates/rwanda/core/section_page.html]:
```suggestion
{% load_sections as sections %}

```

File: gem/templates/rwanda/core/section_page.html [hunk 4]
```diff
@@ -118,7 +118,9 @@ <h2 class=" heading__component">
...
+
```
Comment (sewagodimo) [inline gem/templates/rwanda/core/section_page.html]:
```suggestion
```

File: gem/templates/rwanda/core/section_page.html [hunk 5]
```diff
@@ -118,7 +118,8 @@ <h2 class=" heading__component">
       {% trans "Article categories" %}
     </h2>
   </div>
-  {% load_sections as sections %}
+</div>
```
Comment (sewagodimo) [inline gem/templates/rwanda/core/section_page.html]:
```suggestion
```

**Pull request merged**